front/app/modules/index.ts

```diff
@@ -17,7 +17,7 @@ import intercomConfiguration from './commercial/intercom';
 import satismeterConfiguration from './commercial/satismeter';
 import segmentConfiguration from './commercial/segment';
 import matomoConfiguration from './commercial/matomo';
-import posthogConfiguration from './commercial/posthog';
+import posthogConfiguration from './commercial/posthog_integration';
```

Comment:
Was it working then?

Reply:
Yes, I just renamed this to be consistent with BE engine name